index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Pete Heyes - Developer Chap</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css"/>
+    <link rel="stylesheet" href="css/normalize.css"/>
+    <link href="https://fonts.googleapis.com/css?family=Yellowtail" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+</head>
+<body>
+    <div class="split1">
+        <nav>
+            <ul class="nav-items">
+                <li><a href="#home" class="link">Home</a></li>
```

Comment:
dont need class of link can just use .nav-items li a {} instead

Reply:
this doesnt seem to work but noted for when i make improvements